person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] I will encourage you to make this method private by adding private to the line before the method as required by the project requirement.
![image](https://github.com/clin2on3mun/OOP-School-Library/assets/86200356/75753033-1fe0-420c-ad5f-539110467026)